ProyectoFinal/src/visual/RegEmpresa.java

```diff
@@ -53,211 +62,375 @@ public static void main(String[] args) {
 	/**
 	 * Create the dialog.
 	 */
-	public RegEmpresa() {		
-	setModal(true);
-	setTitle("Registro de Empresa");
-	setBounds(100, 100, 733, 516);
-	setLocationRelativeTo(null);
-	getContentPane().setLayout(new BorderLayout());
-	contentPanel.setBorder(new EmptyBorder(5, 5, 5, 5));
-	getContentPane().add(contentPanel, BorderLayout.CENTER);
-	contentPanel.setLayout(new BorderLayout(0, 0));
-	{
-		JPanel panel = new JPanel();
-		panel.setBorder(new TitledBorder(null, "", TitledBorder.LEADING, TitledBorder.TOP, null, null));
-		contentPanel.add(panel, BorderLayout.CENTER);
-		panel.setLayout(null);
-
-		JLabel lblNewLabel = new JLabel("RNC:");
-		lblNewLabel.setBounds(33, 32, 94, 14);
-		panel.add(lblNewLabel);
-		try{
-			txtFCedCliente = new JFormattedTextField(Utils.getMaskCedula());
-			txtFCedCliente.setForeground(Color.BLACK);
-			txtFCedCliente.setToolTipText("");
-			txtFCedCliente.setBounds(120, 29, 146, 20);
-			panel.add(txtFCedCliente);
-		}catch(Exception ex){}
-		{
-			JLabel lblNewLabel_1 = new JLabel("Tipo de empresa:");
-			lblNewLabel_1.setBounds(382, 115, 98, 14);
-			panel.add(lblNewLabel_1);
-		}
-		{
-		    cbxTipo = new JComboBox();
-			cbxTipo.setModel(new DefaultComboBoxModel(new String[] {"<Seleccione>", "Micro", "Mediana", "Peque\u00F1a ", "Grande", "Emergente"}));
-			cbxTipo.setBounds(502, 112, 156, 20);
-			panel.add(cbxTipo);
-		}
-		{
-			JLabel lblNewLabel_2 = new JLabel("Sector:");
-			lblNewLabel_2.setBounds(382, 73, 98, 14);
-			panel.add(lblNewLabel_2);
-		}
-		{
-			JLabel lblNombre = new JLabel("Nombre Comercial:");
-			lblNombre.setBounds(382, 32, 141, 14);
-			panel.add(lblNombre);
-		}
-		{
-			txtCodigo = new JTextField();
-			txtCodigo.setColumns(10);
-			txtCodigo.setBounds(120, 112, 146, 20);
-			panel.add(txtCodigo);
-		}			
-		{
-			textField = new JTextField();
-			textField.setColumns(10);
-			textField.setBounds(502, 29, 156, 20);
-			panel.add(textField);
-		}
-		{
-			JComboBox cbxSector = new JComboBox();
-			cbxSector.setModel(new DefaultComboBoxModel(new String[] {"<Seleccione>", "Industrial", "Agricultura ", "Alimentaci\u00F3n", "Comercio ", "Construcci\u00F3n", "Educaci\u00F3n ", "Hoteler\u00EDa", "Medios de comunicaci\u00F3n ", "Miner\u00EDa ", "Petrolero ", "Telecomunicaciones ", "Salud ", "Financieros", "P\u00FAblico ", "Silvicultura ", "Textil ", "Tecnol\u00F3gico", "Transporte"}));
-			cbxSector.setBounds(502, 70, 156, 20);
-			panel.add(cbxSector);
-		}
-		{
-			textField_1 = new JTextField();
-			textField_1.setColumns(10);
-			textField_1.setBounds(120, 70, 148, 20);
-			panel.add(textField_1);
-		}
-		{
-			JLabel lblNewLabel_3 = new JLabel("Rubro:");
-			lblNewLabel_3.setBounds(33, 115, 46, 14);
-			panel.add(lblNewLabel_3);
+	public RegEmpresa(Empresa empresa) {	
+		auxEmpresa = empresa;
+		if(auxEmpresa == null ) {
+			setTitle("Registro de Empresa");
+		}else {
+			setTitle("Modificar Empresa");

```

Comment:
```suggestion
			setTitle("Modificar datos de la empresa " + auxEmpresa.getNombreComercial());

```

Para mostrar información relevante en el título de la ventana.